src/mixer/playermanager.h

```diff
@@ -147,17 +147,17 @@ class PlayerManager : public QObject, public PlayerManagerInterface {
 
     // Returns the group for the ith sampler where i is zero indexed
     static QString groupForSampler(int i) {
-        return QString("[Sampler%1]").arg(i+1);
+        return QStringLiteral("[Sampler%1]").arg(i + 1);
     }
 
     // Returns the group for the ith deck where i is zero indexed
     static QString groupForDeck(int i) {
-        return QString("[Channel%1]").arg(i+1);
+        return QStringLiteral("[Channel%1]").arg(i + 1);
```

Comment:
```suggestion
        return QStringLiteral("[Channel") + QString::number(i + 1) + QChar(']');
        
```
How about this?

Reply:
Thought about that but then decided to leave it as is.

I am still unsure what the optimum literals for QStringBuilder are, e.g. QChar vs. QLatin1Char and QStringLiteral vs. QLatin1String? Do we even need all this confusing macro boilerplate when using QStringBuilder?

Reply:
I'm also confused by all the different macros, but if I understand the [wiki page](https://wiki.qt.io/Using_QString_Effectively) correctly, the code above should use 2 or 3 mallocs:
1. Allocate the Channel QSttring
2. Allocate the arg number QString (?)
3. Allocate the final QString

With the QStringBuilfer, it should be 2 mallocs:
1. Allocate QString using QString::number
2. Allocate the final QString

Regarding `QLatin1String` I found [this Qt blog post](https://www.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral) that says:
> **Use QStringLiteral only if a QString needs to be created**
>
> This sounds trivial, but it's not necessarily so. Some methods in the Qt API have overloads for either taking a QString, or a QLatin1String object. This is because Latin1 is simpler to parse than UTF-16, and therefore the QLatin1String version can be faster, and use less memory. This is worth remembering especially for some QString methods:
>
> ```cpp
> bool same = (str == QLatin1String("Hello"));
> str.startsWith(QLatin1String("Hello"));
> str += QLatin1String("World");
> ```
>
>are more efficient than
>
> ```cpp
> bool same = (str == QStringLiteral("Hello"));
> str.startsWith(QStringLiteral("Hello"));
> str += QStringLiteral("World");
> ```

Reply:
I am aware of the dual-typed QString/QLatinString parameters. Unfortunately, if you pass a C string the QString variant will be used implicitly, resulting in an allocation. Therefore I explicitly use QLatin1String("some string") for passing such parameters, e.g. as keys for QJsonObject.

The macros seem to be unnecessary for QStringBuilder expressions.

Reply:
@daschuer I have added the QStringBuilder improvements in `PlayerManager` although they are unrelated and don't belong to this PR.

Reply:
If they don't fit I will revert the changes and can defer them until later.